main/api/src/mill/java9rtexport/Copy.java

```diff
@@ -24,18 +24,17 @@ LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
 NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package io.github.retronym.java9rtexport;
+package mill.java9rtexport;
```

Comment:
Moved this into `mill` so I could make them package private

Reply:
Great!